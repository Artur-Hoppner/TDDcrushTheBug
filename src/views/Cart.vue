<template>
  <div>
    <CartList />
  </div>
</template>

<script>
import CartList from '@/components/CartList/CartList.vue';
export default {
  name: 'Cart',
  components: {
    CartList
  }
};
</script>

<style></style>
