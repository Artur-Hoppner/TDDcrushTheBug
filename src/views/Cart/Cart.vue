<template>
  <div class="cart">
    <CartList />
  </div>
</template>

<script>
import CartList from '@/components/CartList/CartList.vue';
export default {
  name: 'Cart',
  components: {
    CartList
  }
};
</script>

<style lang="scss" scoped>
</style>
