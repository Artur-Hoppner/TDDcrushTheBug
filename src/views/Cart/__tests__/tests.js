import { shallowMount, createLocalVue } from '@vue/test-utils';
import Vuex from 'vuex';
import Cart from '@/views/Cart/Cart.vue';

const localVue = createLocalVue();

localVue.use(Vuex);

describe('Test Cart', () => {
  test('Test if /cart is rendered', () => {
    //Arrange
    const wrapper = shallowMount(Cart, { localVue });
    //Assert
    expect(wrapper.element).toMatchSnapshot();
  });
});
