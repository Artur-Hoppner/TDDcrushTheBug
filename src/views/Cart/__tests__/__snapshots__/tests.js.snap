// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Cart Test if /cart is rendered 1`] = `
<div
  class="cart"
>
  <cartlist-stub />
   
  <!---->
   
  <section
    class="emptyCart"
  >
    <h2>
      
      No items in cart..
      
      <img
        alt="emptyCart"
        class="emptyCartImage"
        src="@/assets/empty-cart.png"
      />
    </h2>
     
    <a
      class="link-to-shop"
      href=""
      id="link-to-shop"
    >
      Check out our products 
      <u>
        here
      </u>
    </a>
    ,
  
  </section>
</div>
`;
